tests/restart/52-cycle-point-time-zone.t

```diff
@@ -46,20 +46,24 @@ run_ok "${TEST_NAME_BASE}-validate" cylc validate "${SUITE_NAME}"
 # Set time zone to +01:00
 export TZ=BST-1
 
-suite_run_ok "${TEST_NAME_BASE}-run" cylc run "${SUITE_NAME}"
+suite_run_ok "${TEST_NAME_BASE}-run" cylc run "${SUITE_NAME}" --hold
+poll_suite_running
```

Comment:
Whenever you do something it pays to check that it has actually been done. When it comes to starting / stopping suites the `poll_suite_*` functions check the filesystem waiting for the condition to be met (for a hardcoded duration before failing).